src/Testcontainers/Containers/ResourceReaper.cs

```diff
@@ -61,8 +61,7 @@ private ResourceReaper(Guid sessionId, IDockerEndpointAuthenticationConfiguratio
         .WithPrivileged(requiresPrivilegedMode)
         .WithAutoRemove(true)
         .WithCleanUp(false)
-        .WithExposedPort(RyukPort)
-        .WithPortBinding(TestcontainersSettings.ResourceReaperPublicHostPort.Invoke(dockerEndpointAuthConfig), RyukPort)
```

Comment:
@kiview Should we remove the delegate assignment immediately? Or should we introduce the change after the next release?

Reply:
Are there any potential risks or potentially breaking behaviour involved in removing it? If you don't see any bigger risks, I think it's fine to remove it for this release already 👍

Reply:
I have added the old code again, which replaces `0` with `string. Empty`. This should behave as intended. In case someone is using the delegate and encounters problems after removing it, we will not break the entire project. They can re-add it. I believe that being cautious in this situation is a bit better. However, I do not expect any issues, as this delegate was introduced in a very early version.